public/app/core/components/TraceToLogs/TraceToLogsSettings.tsx

```diff
@@ -125,24 +125,33 @@ export function TraceToLogsSettings({ options, onOptionsChange }: Props) {
         </InlineField>
       </InlineFieldRow>
 
-      <IntervalInput
-        label={getTimeShiftLabel('start')}
-        tooltip={getTimeShiftTooltip('start')}
-        value={traceToLogs.spanStartTimeShift || ''}
-        onChange={(val) => {
-          updateTracesToLogs({ spanStartTimeShift: val });
-        }}
-        isInvalidError={invalidTimeShiftError}
-      />
-      <IntervalInput
-        label={getTimeShiftLabel('end')}
-        tooltip={getTimeShiftTooltip('end')}
-        value={traceToLogs.spanEndTimeShift || ''}
-        onChange={(val) => {
-          updateTracesToLogs({ spanEndTimeShift: val });
-        }}
-        isInvalidError={invalidTimeShiftError}
-      />
+      <InlineFieldRow>
+        <IntervalInput
+          label={getTimeShiftLabel('start')}
+          tooltip={getTimeShiftTooltip('start')}
+          value={traceToLogs.spanStartTimeShift || ''}
+          onChange={(val) => {
+            updateTracesToLogs({ spanStartTimeShift: val });
+          }}
+          placeholder="0"
+          isInvalidError={invalidTimeShiftError}
+          width={40}
+        />
+      </InlineFieldRow>
+
+      <InlineFieldRow>
+        <IntervalInput
+          label={getTimeShiftLabel('end')}
+          tooltip={getTimeShiftTooltip('end')}
+          value={traceToLogs.spanEndTimeShift || ''}
+          onChange={(val) => {
+            updateTracesToLogs({ spanEndTimeShift: val });
+          }}
+          placeholder="0"
+          isInvalidError={invalidTimeShiftError}
+          width={40}
```

Comment:
I guess we could make the width and placeholder default to these values and save some repetition similar like we have default value for the validation regex.